mgmt/MgmtDefs.h

```diff
@@ -61,6 +63,35 @@ typedef enum {
  */
 typedef void *(*MgmtCallback)(void *opaque_cb_data, char *data_raw, int data_len);
 
+//-------------------------------------------------------------------------
```

Comment:
Added this so the contributions to the HTTP config structures could be easier to work with types compared to `MgmtInt`. It also enables setting the group match by name instead of by enumeration value.